home/whatsapp.py

```diff
@@ -22,16 +25,86 @@ def create_whatsapp_template(name, body, quick_replies=[]):
             buttons.append({"type": "QUICK_REPLY", "text": button})
         components.append({"type": "BUTTONS", "buttons": buttons})
 
+    if image_id:
+        image_handle = upload_image(image_id)
+        components.append(
+            {
+                "type": "HEADER",
+                "format": "IMAGE",
+                "example": {"header_handle": [image_handle]},
+            }
+        )
+
     data = {
         "category": "UTILITY",
         "name": name.lower(),
         "language": "en_US",
         "components": components,
     }
+    response = requests.post(
+        url,
+        headers=headers,
+        data=json.dumps(data, indent=4),
+    )
+    response.raise_for_status()
+
+
+def get_upload_session_id(image_id):
+    url = urljoin(
+        settings.WHATSAPP_API_URL,
+        "graph/v14.0/app/uploads",
+    )
+    headers = {
+        "Content-Type": "application/json",
+    }
+    images = Image.objects.all()
+    for i in images:
+        print(i.id)
```

Comment:
I don't think we need this `print` anymore

Reply:
Yup, removed, thx